src/sentry/static/sentry/app/views/onboarding/createProject.jsx

```diff
@@ -133,70 +117,142 @@ const CreateProject = createReactClass({
-                  {t(
-                    'You cannot create a new project because there are no teams to assign it to.'
-                  )}
-                </PanelAlert>
-                <CreateTeamBody>
+      <React.Fragment>
+        {error && <Alert type="error">{error}</Alert>}
+
+        <div data-test-id="onboarding-info">
+          <PageHeading withMargins>{t('Create a new Project')}</PageHeading>
+          <HelpText>
+            {t(
+              `Projects allow you to scope events to a specific application in
+               your organization. For example, you might have separate projects
+               for your API server and frontend client.`
+            )}
+          </HelpText>
+
+          <PlatformPicker platform={platform} setPlatform={this.setPlatform} showOther />
+          <CreateProjectForm onSubmit={this.createProject}>
+            <div>
+              <FormLabel>{t('Give your project a name')}</FormLabel>
+              <ProjectNameInput>
+                <ProjectPlatformicon monoTone platform={platform} />
+                <input
+                  type="text"
+                  name="name"
+                  label={t('Project Name')}
+                  placeholder={t('Project name')}
+                  autoComplete="off"
+                  value={projectName}
+                  onChange={e => this.setState({projectName: e.target.value})}
+                />
+              </ProjectNameInput>
+            </div>
+            <div>
+              <FormLabel>{t('Assign a Team')}</FormLabel>
+              <TeamSelectInput>
+                <SelectControl
+                  name="select-team"
+                  clearable={false}
+                  value={team}
+                  placeholder={t('Select a Team')}
+                  onChange={val => this.setState({team: val})}
+                  options={teams.map(({slug}) => ({
+                    label: `#${slug}`,
+                    value: slug,
+                  }))}
+                />
+                <Tooltip title={t('Create a team')}>
                   <Button
-                    className="ref-create-team"
-                    priority="primary"
+                    borderless
+                    type="button"
+                    icon="icon-circle-add"
                     onClick={() =>
                       openCreateTeamModal({
                         organization,
+                        onClose: ({slug}) => this.setState({team: slug}),
                       })}
-                  >
-                    {t('Create a Team')}
-                  </Button>
-                </CreateTeamBody>
-              </React.Fragment>
-            }
-          />
-        )}
-      </div>
+                  />
+                </Tooltip>
+              </TeamSelectInput>
+            </div>
+            <div>
+              <Button
+                priority="primary"
+                disabled={inFlight || !team || projectName === ''}
```

Comment:
can we create a default team if they don't create one? we wanna reduce the number of steps / clicks in the "happy path" as much as possible. since this is their first experience of sentry, making administrative work like this mandatory seems less than ideal?

Reply:
They have a default team when they create their org. They will only not have a team if they deleted it.